packages/nitro-protocol/ganache/index.ts

```diff
@@ -0,0 +1,36 @@
+import {configureEnvVariables, GanacheServer} from '@statechannels/devtools';
```

Comment:
Did you put any thought into whether `nitro-protocol/ganache` was a good location for this? I did not put much thought into the name `ganache`, and I'm not sure it's a good name.

Maybe `nitro-protocol/dev-environment` is better? Or, maybe it even belongs in its own package.

Reply:
Wasn't too concerned with the name of it as the naming can always just change. Just wanted to get things up and running.